src/main/java/com/myjeeva/digitalocean/pojo/Project.java

```diff
@@ -0,0 +1,125 @@
+package com.myjeeva.digitalocean.pojo;
+
+import com.google.gson.annotations.Expose;
+import com.myjeeva.digitalocean.common.Environment;
+import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
+
+/**
+ * Represents Project attributes of DigitalOcean.
+ *
+ * @author Thomas Crombez (thomasc@trikthom.com)
+ */
+public class Project extends Base {
+
+//  private static final long serialVersionUID = ??;
+
+  private String id;
+
+  private String owner_uuid;
```

Comment:
@TrikThom Please do not use the snake case in Java, kindly go with camel case for the variable name. Use JSON annotation for field mapping into a variable like other POJO.

Kindly take care of all variables.